graphql/util.go

```diff
@@ -22,7 +22,11 @@ func GetGroupedFiles(ctx context.Context, name string, taskID string, execution
 		return nil, ResourceNotFound.Send(ctx, err.Error())
 	}
 	hasUser := gimlet.GetUser(ctx) != nil
-	strippedFiles := artifact.StripHiddenFiles(taskFiles, hasUser)
+	strippedFiles, err := artifact.StripHiddenFiles(taskFiles, hasUser)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, err.Error())
```

Comment:
will this leak the filename if the file doesn't contain signing params? (and does it matter if it does)

Reply:
Interesting. 
If it does matter, I can remove the filename from here: 
return nil, errors.Errorf("error presigning the url for %s, awsSecret, awsKey, bucket, or filekey missing", file.Name)
Maybe John will have insight into whether or not it matters.

Reply:
At the moment it doesn't matter, but eventually not logged-in users should be able to get here so let's not send up the error

Reply:
I changed it to return err instead of using InternalServerError.Send. Is that better?

Reply:
What I meant is to return an error that you type yourself, rather passing along the error from that function

Reply:
That makes sense. Does it look okay now?

Reply:
Looks good!